SETUP/document_database.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+$relPath='../pinc/';
+include_once($relPath.'base.inc');
+include_once($relPath.'misc.inc');
+include_once($relPath . 'TableDocumentation.inc');
+
+// First is the script name, the second is the operation name
+$number_of_arguments = $argc - 2;
+
+if ($argc < 2) {
+    echo "No operation was chosen.\n";
+    echo "Supported operations are: generate\n";
+
+    exit(1);
+}
+
+// generate <directory_path> <table_name or all>
+if ($argv[1] === 'generate') {
+    if ($argc !== 4) {
+        echo "Operation generate requires 2 arguments, $number_of_arguments were given.\n";
+        echo "Supported syntax for generate command is 'generate <directory path> <table name or all>'.\n";
+
+        exit(1);
+    }
+
+    $directory_path = $argv[2];
+    $table_name = $argv[3];
+
+    if (!is_dir($directory_path)) {
+        echo "File path '$directory_path' does not exist or is not a directory.\n";
+
+        exit(1);
+    }
+
+    if ($table_name === 'all') {
+        generate_files_for_all_tables($directory_path);
+    }
+    else {
+        generate_file_for_table($table_name, $directory_path . '/' . $table_name . '.md', $table_name);
```

Comment:
When creating filenames and paths, the rest of the code (or at least the bulk of it) does it with string interpolation, eg:
```
        generate_file_for_table($table_name, "$directory_path/$table_name.md", $table_name);
```
Lets continue to do that in this file too please.